apps/accounts/tasks.py

```diff
@@ -1,14 +1,21 @@
 from celery import shared_task
+from celery.worker import request
+
 from django.core.mail import EmailMessage
+
 from django.template.loader import render_to_string
 
+from apps.accounts.models import User
+
 
 @shared_task
-def send_verification_email(user, domain):
+def send_verification_email(user_id):
+    user = User.objects.get(id=user_id)
     subject = 'MyProjectSet email verification'
+
     message = render_to_string('accounts/activate_account.html', {
-        'user': user,
-        'domain': domain,
+        'email': user.email,
+        'domain': 'lysenok.planeks.net',
```

Comment:
Do not hardcode the domain in the code. Use settings and environment variables instead.